ocs_ci/utility/uninstall_openshift_logging.py

```diff
@@ -0,0 +1,53 @@
+"""
+Function to teardown the openshift-logging
+"""
+import logging
+
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.ocs.resources.pod import get_all_pods, verify_pod_exists
+
+logger = logging.getLogger(__name__)
+
+
+def uninstall_cluster_logging():
+    """
+    Function to uninstall cluster-logging from the cluster
+    1. Deletes the project "openshift-logging"
+        & "openshift-operators-redhat"
+    2. Deletes the CSC resource "cluster-logging-operator"
```

Comment:
'2' is achieved by deleting the openshift_operators_redhat project I believe. can you update the doc string accordingly?

Reply:
Done